homeassistant/components/smartthings/sensor.py

```diff
@@ -749,6 +760,55 @@ class SmartThingsSensorEntityDescription(SensorEntityDescription):
             )
         ],
     },
+    # Added - Samsung Washer
+    Capability.SAMSUNG_CE_WASHER_OPERATING_STATE: {
+        Attribute.OPERATING_STATE: [
+            SmartThingsSensorEntityDescription(
+                key=Attribute.OPERATING_STATE,
+                translation_key="washer_operating_state",
+                icon="mdi:state-machine",
+                device_class=SensorDeviceClass.ENUM,
+                options_attribute=Attribute.SUPPORTED_OPERATING_STATES,
+            )
+        ],
+        Attribute.OPERATION_TIME: [
+            SmartThingsSensorEntityDescription(
+                key=Attribute.OPERATION_TIME,
+                translation_key="washer_operation_time",
+                native_unit_of_measurement=UnitOfTime.MINUTES,
+                icon="mdi:clock",
+            )
+        ],
+        Attribute.PROGRESS: [
+            SmartThingsSensorEntityDescription(
+                key=Attribute.PROGRESS,
+                translation_key="washer_operating_progress",
+                native_unit_of_measurement=PERCENTAGE,
+                icon="mdi:progress-clock",
+            )
+        ],
+        Attribute.REMAINING_TIME: [
+            SmartThingsSensorEntityDescription(
+                key=Attribute.REMAINING_TIME,
+                translation_key="washer_remaining_time",
+                native_unit_of_measurement=UnitOfTime.MINUTES,
+                icon="mdi:timelapse",
+            )
+        ],
+    },
+    # Added - Samsung Washer
+    Capability.SAMSUNG_CE_WATER_CONSUMPTION_REPORT: {
+        Attribute.WATER_CONSUMPTION: [
+            SmartThingsSensorEntityDescription(
+                key=Attribute.WATER_CONSUMPTION,
+                translation_key="water_consumption",
+                state_class=SensorStateClass.TOTAL_INCREASING,
+                device_class=SensorDeviceClass.WATER,
+                native_unit_of_measurement="L",
+                value_fn=lambda value: value["cumulativeAmount"] / 1000,
+            )
+        ]
+    },
```

Comment:
I think these would be nice in a separete PR as well as we need to learn more about this capability